tests/baselines/reference/capturedLetConstInLoop3.js

```diff
@@ -273,6 +273,7 @@ function foo2(x) {
 function foo3(x) {
     var _loop_5 = function () {
         var x_5;
+        var v;

```

Comment:
The original code was:

```
function foo3(x) {
    do {
        let x;
        var v;
        (function() { return x + v });
        (() => x + v);
    } while (1 === 1);
    
    use(v);
}
```

But this just declared an all new `v`?

Reply:
shoot, I missed that those were hoisted to the outer scope already here so this created a bug. I added an extra test case for that bug and fixed this.